### PR TITLE
Charts

### PR DESCRIPTION
**Pull request by kacpergumieniuk**
## Summary by CodeRabbit

- **New Features**
  - Introduced a new chart component to visualize market data within TradingHub.
  - Implemented dynamic chart data fetching and rendering based on the selected market.

- **Updates**
  - Enhanced TradingHub with improved data fetching logic and clearer function definitions.
  - Updated CSS for better styling consistency across the TradingHub component.

- **Bug Fixes**
  - Fixed inconsistencies in data fetching and rendering for market charts.

### CODE REVIEW

**Comment by Klapeyron** [comment]
Chart is not visible if wallet is not connected

File: types/chartTypes.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export interface ChartFeedType {
+  createPrice: BigInt;
```
[the remark this answers is not in the document]
Comment (kacpergumieniuk) [reply types/chartTypes.ts:2]: im using pascal case

File: components/Chart/Chart.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      }
+    };
+
+    updateDimensions();
+    window.addEventListener('resize', updateDimensions);
+
+    return () => {
+      window.removeEventListener('resize', updateDimensions);
+    };
+  }, []);
```
[the remark this answers is not in the document]
Comment (kacpergumieniuk) [reply components/Chart/Chart.tsx]: outdated

File: types/chartTypes.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (kacpergumieniuk) [reply types/chartTypes.ts:2]: okay then, i will consistently change primitive types to lowercase

**Pull request merged**